manual/genManual.cpp

```diff
@@ -319,8 +319,14 @@ static void writeOutput()
       //
       QString out;
       addHeader(out);
-      out += "<h2>Score Elements</h2>\n"
-             "<ul>\n";
+      out += "<h2>Score Elements</h2>\n";
+      out += "<h3>Quick Guide</h3>\n";
+      out += "<p>Below are all the various classes you can use."
+             "<br>The main class is <a href='musescore.html'>MuseScore</a>."
+             "<br>Use 'New' to create a skeleton plugin."
+             "<br>Plugins are coded in <a href='http://doc.qt.io/qt-5/qmlapplications.html#what-is-qml'>QML</a>"
```

Comment:
Not sure, do we want these strings to be translatable?

Reply:
Well, it wasn't translatable before...

Reply:
Do you just want me to surround it with a "tr"?

Reply:
As said, I'm not sure whether we want or need ot translatable. @lasconic?

Reply:
Yeah, that's in genmanual.cpp, and there are no "tr" features at all in there... I think that counts as a whole extra issue.

Reply:
Yes, surely not your fault. But might make up yet another improvement. I'm just not sure whether it is worth the effort?

Reply:
Well, the issue is that that plugin help auto generates metadata in the header files... which are not translated.
So it's a whole new kettle of fish. Worth thinking about, but above and beyond the current context, I think,

Reply:
Agreed